lib/libs/webforms/CS8/v202401.ts

```diff
@@ -0,0 +1,690 @@
+                    required: "* Required",
+                  },
+                  props: {
+                    options: [
+                      {
+                        label: "From age 19, up to specific age",
+                        value: "from-age-19",
+                        slots: [
+                          {
+                            rhf: "Input",
+                            name: "end-age-range",
+                            rules: {},
+                            label: "End of age range",
+                            labelClassName: "font-bold",
+                            props: {
+                              className: "w-[125px]",
+                            },
+                          },
+                        ],
+                      },
+                      {
+                        label: "No age restriction",
+                        value: "no-age-restriction",
+                        slots: [
+                          {
+                            rhf: "Textarea",
+                            name: "no-age-describe-applicant-child-preg-woman",
+                            rules: {
+                              required: "* Required",
+                              pattern: {
+                                value: noLeadingTrailingWhitespace,
+                                message:
+                                  "Must not have leading or trailing whitespace.",
+                              },
+                            },
+                            label:
+                              "Describe how it’s determined whether the applicant will be provided coverage as a child or as a pregnant woman.",
+                            labelClassName: "font-bold",
+                            props: {
+                              className: "w-[696px]",
+                            },
+                          },
+                        ],
+                      },
+                      {
+                        label: "Another age range",
+                        value: "another-age-range",
+                        slots: [
+                          {
+                            rhf: "WrappedGroup",
+                            name: "wrapped",
+                            props: {
+                              wrapperClassName: "flex-row flex gap-5",
+                            },
+                            fields: [
+                              {
+                                rhf: "Input",
+                                name: "start-age-range",
+                                rules: {
+                                  pattern: {
+                                    value: /^\d*\.?\d+$/,
+                                    message: "Must be a positive percentage",
+                                  },
+                                  required: "* Required",
+                                },
+                                label: "Start of age range",
```

Comment:
Should this include validation to ensure the "End of age range" value is greater than "Start of age range"? Don't know if it's a requirement or not

Reply:
Good point let me look at our other examples